### PR TITLE
Exclude and include paths.

### PR DESCRIPTION
**Pull request by fhwang**

### CODE REVIEW

File: lib/cc/engine/csslint.rb [hunk 1]
```diff
@@ -48,14 +48,31 @@ def results
...
       end
 
+      def build_files_with_exclusions(exclusions)
+        files = Dir.glob("**/*.css")
+        files.reject { |f| exclusions.include?(f) }
+      end
+
+      def build_files_with_inclusions(inclusions)
+        inclusions.map do |include_path|
+          if include_path =~ %r{/$}
```
Comment (GordonDiggs) [inline lib/cc/engine/csslint.rb]: this might be more readable as `include_path.ends_with?("/")`. WDYT?

Comment (fhwang) [inline lib/cc/engine/csslint.rb]: It would be, but we're not using activesupport here and keeping things light might be a higher priority here.

Comment (GordonDiggs) [inline lib/cc/engine/csslint.rb]: Oh good call.

**Comment by GordonDiggs** [comment]
@fhwang re: the CC errors. We probably want to import a `rubocop,yml` from another repo and use it here

**Comment by fhwang** [comment]
@GordonDiggs sure, any thoughts on which other repo?

**Comment by GordonDiggs** [comment]
@fhwang this one looks good https://github.com/codeclimate/codeclimate/blob/master/.rubocop.yml. Looks like we [also exclude spec](https://github.com/codeclimate/codeclimate/blob/master/.codeclimate.yml#L10-L12)

**Comment by fhwang** [comment]
@GordonDiggs take a look now?

**Comment by GordonDiggs** [comment]
LGTM

**Pull request merged**